CHANGELOG.md

```diff
@@ -62,6 +62,7 @@
 * Small performance improvements for radiation
 * Small performance improvements for Auto Disenchanters
 * Magnesium Salt in Magnesium-Salt generators now lasts longer
+* Performance improvements for TickerTask(Slimefun ticking)
```

Comment:
Please remove this from the change log.
There is no way such a drastic change, especially when it's about threading and concurrency... will make it into the next RC.